app/cypress/e2e/home.cy.ts

```diff
@@ -180,9 +147,33 @@ describe('Home Page', () => {
     cy.wait(2000);
     cy.findByText('Continue').should('exist').click();
 
+    cy.selectProfile('Assessment Units');
+    cy.findByRole('button', { name: 'Advanced Queries' }).click();
+
     cy.selectCopyBox(
       'api-query-copy-box-container',
       `${origin}/api/attains/assessmentUnits?${columnsValue}&assessmentUnitStatus=A&format=tsv&api_key=<YOUR_API_KEY>`,
     );
   });
+
+  it('Tooltip displays when hovering over info icon', () => {
+    cy.selectProfile('Assessment Units');
+
+    cy.findByRole('button', { name: 'Region tooltip' }).focus();
+    cy.findByText(
+      'The geographic region or area to which a particular Assessment Unit belongs.',
+    ).should('be.visible');
+  });
+
+  it.only('Source select menu filters corresponding options', () => {
```

Comment:
```suggestion
  it('Source select menu filters corresponding options', () => {
```

Can you remove the `only` flag?